src/Microsoft.Sbom.Api/Config/ApiConfigurationBuilder.cs

```diff
@@ -123,9 +123,10 @@ public static InputConfiguration GetConfiguration(
             throw new ArgumentException($"'{nameof(outputPath)}' cannot be null or whitespace.", nameof(outputPath));
         }
 
+        // TODO: update to SPDX 3.0 for default.
```

Comment:
TODO in the future, not in this PR, correct?

Reply:
Yes, this is for when we eventually default to generating spdx 3.0 SBOMs instead of 2.2. For now, the default is still 2.2.